libpod/options.go

```diff
@@ -130,6 +130,20 @@ func WithConmonPath(path string) RuntimeOption {
 	}
 }
 
+// WithSyslog specifies whether the OCI RUntime should write to syslog
+// runtime.
```

Comment:
Is this part of the previous sentence?  I would have expected it to stop with "should write to syslog.".

Reply:
Fixed